tool/planet/udev.go

```diff
@@ -112,30 +114,30 @@ func (r *udevListener) createDevice(device *configs.Device) error {
 		return trace.Wrap(err)
 	}
 
-	err = enter(deviceCmd("add", "--data", string(deviceJson)))
+	err = enter(deviceCmd(r.seLinux, "add", "--data", string(deviceJson)))
```

Comment:
Nit: I'd just make `deviceCmd` a udevListener method so you don't have to pass this new parameter.